Dockerfile

```
@@ -34,20 +36,30 @@ RUN git clone --recursive --depth 1 --branch $PHP_ZSTD_VERSION https://github.co
   && ./configure --with-libzstd \
   && make && make install
 
+## LZ4 Extension
+FROM compile AS lz4
+RUN git clone --recursive --depth 1 --branch $PHP_LZ4_VERSION https://github.com/kjdev/php-ext-lz4.git \
+  && cd php-ext-lz4 \
```

And this is how you check out a specific commit:
```suggestion
RUN git clone --recursive --depth 1 https://github.com/kjdev/php-ext-lz4.git \
  && cd php-ext-lz4 \
  && git reset --hard $PHP_LZ4_COMMIT \
```